django/test/testcases.py

```diff
@@ -175,13 +175,15 @@ def setUpClass(cls):
             for alias in connections:
                 connection = connections[alias]
                 connection.cursor = _CursorFailure(cls.__name__, connection.cursor)
+                connection.chunked_cursor = _CursorFailure(cls.__name__, connection.chunked_cursor)
```

Comment:
Is a regression test needed for this change? I don't see any failures after reverting the changes to this file.